spec/requests/home_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
- [ ] Kindly make sure that all tests are passed, now there are 4 failed tests.
![image](https://github.com/lorenzoworx/chef-chapters/assets/24703171/6cda956f-4ad8-4e29-9ff8-4b55d4f260b4)